framework/source/class/qx/test/log/Logger.js

```diff
@@ -89,16 +89,24 @@ qx.Class.define("qx.test.log.Logger",
       qx.log.Logger.clear();
       qx.log.Logger.register(appender);
 
+      qx.Class.define("test.DisposableObject", {
```

Comment:
could you please undefine the class via ```qx.Class.undefine``` to prevent pollution with test classes? Thank you.